File: reverse-words-in-a-string-iii/reverse-words-in-a-string-iii.py
class Solution:
    def reverseWords(self, s: str) -> str:
        word_list = s.split(' ') # Make sentence mutable
        word_list = [[ch for ch in word]for word in word_list] # Make chars mutable
        word_list = [word[::-1] for word in word_list] # Reverse the words
        word_list = [''.join(word) for word in word_list] # Recombine words
        return ' '.join(word_list) # Return combined sentence

File: reverse-words-in-a-string-iii/reverse-words-in-a-string-iii.java
class Solution 
{
    public String reverseWords(String s) 
    {
        // Split the words
        String words[] = s.split(" "); 
        
        // Reverse each word and concatenate into string builder
        StringBuilder answer = new StringBuilder();
        for (String word : words)
            answer.append(new StringBuffer(word).reverse().toString() + " ");
        return answer.toString().trim(); // Return reformatted string
    }
}